framework/generated/generated_vulkan_state_table.h

```diff
@@ -313,6 +313,375 @@ class VulkanStateTable : VulkanStateTableBase
+    std::unordered_map<VkDescriptorPool, DescriptorPoolWrapper*> descriptorPool_map_;
+    std::unordered_map<VkDescriptorSet, DescriptorSetWrapper*> descriptorSet_map_;
+    std::unordered_map<VkDescriptorSetLayout, DescriptorSetLayoutWrapper*> descriptorSetLayout_map_;
+    std::unordered_map<VkDescriptorUpdateTemplate, DescriptorUpdateTemplateWrapper*> descriptorUpdateTemplate_map_;
+    std::unordered_map<VkDevice, DeviceWrapper*> device_map_;
+    std::unordered_map<VkDeviceMemory, DeviceMemoryWrapper*> deviceMemory_map_;
+    std::unordered_map<VkDisplayKHR, DisplayKHRWrapper*> displayKHR_map_;
+    std::unordered_map<VkDisplayModeKHR, DisplayModeKHRWrapper*> displayModeKHR_map_;
+    std::unordered_map<VkEvent, EventWrapper*> event_map_;
+    std::unordered_map<VkFence, FenceWrapper*> fence_map_;
+    std::unordered_map<VkFramebuffer, FramebufferWrapper*> framebuffer_map_;
+    std::unordered_map<VkImage, ImageWrapper*> image_map_;
+    std::unordered_map<VkImageView, ImageViewWrapper*> imageView_map_;
+    std::unordered_map<VkIndirectCommandsLayoutNV, IndirectCommandsLayoutNVWrapper*> indirectCommandsLayoutNV_map_;
+    std::unordered_map<VkInstance, InstanceWrapper*> instance_map_;
+    std::unordered_map<VkMicromapEXT, MicromapEXTWrapper*> micromapEXT_map_;
+    std::unordered_map<VkOpticalFlowSessionNV, OpticalFlowSessionNVWrapper*> opticalFlowSessionNV_map_;
+    std::unordered_map<VkPerformanceConfigurationINTEL, PerformanceConfigurationINTELWrapper*> performanceConfigurationINTEL_map_;
+    std::unordered_map<VkPhysicalDevice, PhysicalDeviceWrapper*> physicalDevice_map_;
+    std::unordered_map<VkPipeline, PipelineWrapper*> pipeline_map_;
+    std::unordered_map<VkPipelineCache, PipelineCacheWrapper*> pipelineCache_map_;
+    std::unordered_map<VkPipelineLayout, PipelineLayoutWrapper*> pipelineLayout_map_;
+    std::unordered_map<VkPrivateDataSlot, PrivateDataSlotWrapper*> privateDataSlot_map_;
+    std::unordered_map<VkQueryPool, QueryPoolWrapper*> queryPool_map_;
+    std::unordered_map<VkQueue, QueueWrapper*> queue_map_;
+    std::unordered_map<VkRenderPass, RenderPassWrapper*> renderPass_map_;
+    std::unordered_map<VkSampler, SamplerWrapper*> sampler_map_;
+    std::unordered_map<VkSamplerYcbcrConversion, SamplerYcbcrConversionWrapper*> samplerYcbcrConversion_map_;
+    std::unordered_map<VkSemaphore, SemaphoreWrapper*> semaphore_map_;
+    std::unordered_map<VkShaderModule, ShaderModuleWrapper*> shaderModule_map_;
+    std::unordered_map<VkSurfaceKHR, SurfaceKHRWrapper*> surfaceKHR_map_;
+    std::unordered_map<VkSwapchainKHR, SwapchainKHRWrapper*> swapchainKHR_map_;
+    std::unordered_map<VkValidationCacheEXT, ValidationCacheEXTWrapper*> validationCacheEXT_map_;
+    std::unordered_map<VkVideoSessionKHR, VideoSessionKHRWrapper*> videoSessionKHR_map_;
+    std::unordered_map<VkVideoSessionParametersKHR, VideoSessionParametersKHRWrapper*> videoSessionParametersKHR_map_;
```

Comment:
Re my question above, are we maintaining this set of maps live in the system at the same time as the `std::map` equivalents above?